Source/Node/src/MFRC522Reader.cpp

```diff
@@ -0,0 +1,112 @@
+#include "MFRC522Reader.hpp"
+#include <SPI.h>
+
+
+#define DEFAULT_SS_PIN 10
+#define DEFAULT_RST_PIN 9
+#define DEFAULT_KEY 0xFF
+#define DEFAULT_TRAILER_BLOCK 7
+
+MFRC522Reader::MFRC522Reader()
```

Comment:
Why not call the other constructor? This code is the same except that the SS and RST pins are using the default values. Or maybe only 1 constructor with default parameter values.